packit/distgit.py

```diff
@@ -407,10 +407,20 @@ def create_bodhi_update(
         rendered_note = update_notes.format(version=self.specfile.get_full_version())
         try:
             result = b.save(builds=koji_builds, notes=rendered_note, type=update_type)
-            logger.info(f"Bodhi response:\n{result}")
+            logger.debug(f"Bodhi response:\n{result}")
+            logger.info(
+                f"Bodhi update {result['alias']}:\n"
+                f"- {result['url']}\n"
+                f"- autokarma: {result['autokarma']}\n"
+                f"- stable_karma: {result['stable_karma']}\n"
+                f"- unstable_karma: {result['unstable_karma']}"
```

Comment:
and maybe even notes? so that it's clear what the update is about

Reply:
fixed